tests/functional/test_basics.py

```diff
@@ -220,6 +220,49 @@ def test_recursive_and_batch_exclusive(run_line, option):
     assert f"You cannot use `{option}` in addition to `--batch`" in result.stderr
 
 
+@pytest.mark.parametrize(
+    "deletion_option,recursion_option,expected_error",
+    (
+        ("--delete-destination-extra", "--recursive", ""),
+        ("--delete-destination-extra", "", ""),
+        (
+            "--delete-destination-extra",
+            "--no-recursive",
+            (
+                "The `--delete-destination-extra` option cannot be specified with "
+                "`--no-recursive`."
+            ),
+        ),
+        ("--delete", "--recursive", ""),
+        ("--delete", "", ""),
+        (
+            "--delete",
+            "--no-recursive",
+            "The `--delete` option cannot be specified with `--no-recursive`.",
+        ),
+    ),
+)
+def test_no_recursive_and_delete_exclusive(
+    run_line,
+    deletion_option,
+    recursion_option,
+    expected_error,
+):
+    load_response("transfer.get_submission_id")
+    load_response("transfer.submit_transfer")
+    ep_meta = load_response("transfer.get_endpoint").metadata
+    ep_id = ep_meta["endpoint_id"]
```

Comment:
Not entirly sure if I'm using `load_response` correctly here, I was basing this off a similar test: `test_trasfer_create::test_timer_creation_delete_flag_requires_recursion`